src/gui/src/gui.tcl

```diff
@@ -170,6 +171,24 @@ proc select { args } {
     set name $keys(-name)
   }
   
+  set attribute ""
+  set value ""
+  if { [info exists keys(-filter)] } {
+    if { $name != "" } {
+      utl::error GUI 59 "-filter not available when using -name."
+    } else {
+      set filter $keys(-filter)
+      set filter [split $filter "="]
+      set length [llength $filter]
+      if { $length != 2 } {
+        utl::error GUI 56 "Invalid syntax for -filter. Use -filter attribute=value."
```

Comment:
It think we could eventually allow a list of key=value filters that would be and'ed together.  Let's revisit that once we get the single case working.